community/contribute/scholarships.md

```diff
@@ -0,0 +1,13 @@
+# Scholarships
+
+The [PEP-31 Scholarship Fund](https://forum.pokt.network/t/pep-31-scholarship-fund/) was [approved](https://gov.pokt.network/#/proposal/0x2920a4057b4aa6af033654bfcfdd77f2b98c81f100d8e55b97a30101e563db46) by the DAO in 06/2022 to allocate an accumulating monthly allowance of `10,000/POKT` per month to contributors, students, learners, speakers, educators and others or in the Pocket community.
+
+## Recipients
+
+### June 2022
+
+Tyler Kell ([@relyt29](https://twitter.com/relyt29)) is the first recipient of the Pocket Scholarship for his presentation `Academic Cryptocurrency Research Space - An Overview, Spring 2022` at [Infracon](https://www.infracon.org/) 2022 - Pocket's inaugral conference.
```

Comment:
I'd like to have as few binary files under version control as possible, especially one that's ~4MB. Is there any other place this file is hosted that we could link to?

Reply:
Ended up uploading it to git lfs instead. Is that fine with you?

Reply:
Brilliant. Definitely a great solution.